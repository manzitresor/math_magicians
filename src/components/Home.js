import React from 'react';
import styles from '../styles/Home.module.css';
import magician from '../assets/images.png';

function Home() {
  return (
    <>
      <div className={styles.mainSection}>
        <div className={styles.magician}>
          <img src={magician} alt="magician" className={styles.magicianImage} />
        </div>
        <div className={styles.welcome}>
          <h1>welcome to our home page</h1>
          <p className={styles.paragraph}>
            Founded in 2020, Mathmagicians focuses on teaching competition math
            to middle school students or below.
            We hope to help students do well in competitions and inspire
            them to keep learning. We are actively looking to expand our team of tutors to
            better serve the students and give back to the community.
          </p>
        </div>
      </div>

    </>
  );
}

export default Home;
