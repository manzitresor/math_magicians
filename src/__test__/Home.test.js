import React from 'react';
import renderer from 'react-test-renderer';
import { render, screen } from '@testing-library/react';
import Home from '../components/Home';

describe('Home component', () => {
  test('renders home component', () => {
    render(<Home />);
    screen.debug();
  });

  test('renders correctly', () => {
    const component = renderer.create(
      <Home />,
    );
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
