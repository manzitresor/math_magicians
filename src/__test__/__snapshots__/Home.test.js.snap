// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home component renders correctly 1`] = `
<div
  className="mainSection"
>
  <div
    className="magician"
  >
    <img
      alt="magician"
      className="magicianImage"
      src="images.png"
    />
  </div>
  <div
    className="welcome"
  >
    <h1>
      welcome to our home page
    </h1>
    <p
      className="paragraph"
    >
      Founded in 2020, Mathmagicians focuses on teaching competition math to middle school students or below. We hope to help students do well in competitions and inspire them to keep learning. We are actively looking to expand our team of tutors to better serve the students and give back to the community.
    </p>
  </div>
</div>
`;
